packages/a/anki/files/net.ankiweb.Anki.appdata.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>net.ankiweb.Anki</id>
+  <translation type="gettext">anki</translation>
+  <name>Anki</name>
+  <summary>Powerful, intelligent flash cards</summary>
+  <developer_name>Damien Elmes</developer_name>
+  <description>
+    <p>Anki is a program which makes remembering things easy. Because it's a lot more efficient than traditional study methods, you can either greatly decrease your time spent studying, or greatly increase the amount you learn.</p>
+    <p>Anyone who needs to remember things in their daily life can benefit from Anki. Since it is content-agnostic and supports images, audio, videos and scientific markup (via LaTeX), the possibilities are endless.</p>
+    <p>Optional LaTeX support is provided by the TeX Live Flatpak extension org.freedesktop.Sdk.Extension.texlive.</p>
+  </description>
+  <kudos>
+    <kudo>ModernToolkit</kudo>
+    <kudo>UserDocs</kudo>
+  </kudos>
+  <project_license>AGPL-3.0+</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+  <url type="homepage">https://apps.ankiweb.net/</url>
+  <url type="bugtracker">https://forums.ankiweb.net/</url>
+  <url type="help">https://docs.ankiweb.net/</url>
+  <url type="donation">https://apps.ankiweb.net/support/</url>
+  <url type="translate">https://translating.ankiweb.net/</url>
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">https://iili.io/HU8RpwX.png</image>
+      <caption>Flashcard and answer</caption>
+    </screenshot>
+  </screenshots>
+  <content_rating type="oars-1.1"/>
+  <provides>
+    <id>net.ankiweb.Anki.desktop</id>
+  </provides>
+  <launchable type="desktop-id">anki.desktop</launchable>
+  <update_contact>dan_at_reactivated.net</update_contact>
```

Comment:
Should this be included? I'd hate to have that shown to people when they should be coming to us to ask for updates. Unfortunately, I can't find any documentation on what that key is supposed to be for.

Reply:
Hmmm, I can remove that. It doesn't affect validation. 
It barely shows in GS, It should be grouped together with the other URL in middle right of app page, just above comments.